RELEASES.md

```diff
@@ -0,0 +1,248 @@
+
+After a minor release, a branch will be created, with the format
+`release/<major>.<minor>` from the minor tag. All further patch releases will
+be done from that branch. For example, once we release `v1.0.0`, a branch
+`release/1.0` will be created from that tag. All future patch releases will be
+done against that branch.
+
+### Pre-releases
+
+Pre-releases, such as alphas, betas and release candidates will be conducted
+from their source branch. For major and minor releases, these releases will be
+done from master. For patch releases, these pre-releases should be done with in
+the corresponding release branch.
+
+While pre-releases are done to assist in the stabilization process, no
+guarantees are provided.
+
+### Upgrade Path
+
+The upgrade path for containerd is such that the 0.0.x patch releases are
+always backward compatible with its major and minor version. Minor (0.x.0)
+version will always be compatible with the previous minor release. i.e. 1.2.0
+is backwards compatible with 1.1.0 and 1.1.0 is compatible with 1.0.0. There is
+no compatibility guarantees with upgrades from two minor releases. i.e. 1.0.0
+to 1.2.0.
+
+There are not backwards compatibility guarantees with upgrades to major
+versions. i.e 1.0.0 to 2.0.0. Each major version will be supported for 1 year
+with bug fixes and security patches.
+
+### Next Release
+
+The activity for the next release will be tracked in the
+[milestones](https://github.com/containerd/containerd/milestones). If your
+issue or PR is not present in a milestone, please reach out to the maintainers
+to create the milestone or add an issue or PR to an existing milestone.
+
+### Support Horizon
+
+We'll discuss the support horizon a `minor` release in the context of the
+corresponding release branch, identified by `<major>.<minor>`. Releases
+branches will be in one of several states:
+
+- __*Next*__: The next planned release branch.
+- __*Active*__: The release is currently supported and accepting patches.
+- __*End of Life*__: The release branch is no longer support and no new patches will be accepted.
+
+Releases will be supported up to one year after a _minor_ release. This means that
+we will accept bug reports and backports to release branches until the end of
+life date. If no new _minor_ release has been made, that release will be
+considered supported until the next _minor_ is released or one year, whichever
+is longer.
+
+The current state is available in the following table:
+
+| Release | Status      | Start            | End of Life       |
+|---------|-------------|------------------|-------------------|
+| [0.0](https://github.com/containerd/containerd/releases/tag/0.0.5)  | End of Life | Dec 4, 2015  | - |
+| [0.1](https://github.com/containerd/containerd/releases/tag/v0.1.0) | End of Life | Mar 21, 2016 | - |
+| [0.2](https://github.com/containerd/containerd/tree/v0.2.x)         | Active | Apr 21, 2016      | Upon 1.0 release |
+| 1.0     | Next   | TBD  | TBD+1 year |
```

Comment:
By my reading oft he preceding paragraphs EOL for 1.0 is actually `max(TBD+1 year, release of 1.1.0)`?